requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Can you re-run this with Python 3.8? sometimes packages have different dependencies at different versions and we're currently building most Open edX repositories with Python 3.8.

Reply:
updated! learned about `pyenv` and used `3.8`!